modulemd/v2/include/modulemd-2.0/private/modulemd-module-stream-v1-private.h

```diff
@@ -30,4 +30,39 @@ G_BEGIN_DECLS
  */
 
 
+/**
+ * modulemd_module_stream_v1_parse_yaml:
+ * @parser: (inout): A yaml parser positioned at the start of the data subsection
+ * of a modulemd v1 YAML document.
+ * @error: (out): A #GError that will return the reason for a parsing or
+ * validation error.
+ *
+ * Returns: (transfer full): A newly-allocated #ModulemdModuleStreamV1 object
+ * read from the YAML. NULL if a parse or validation error occurred and sets
+ * @error appropriately.
+ *
+ * Since: 2.0
+ */
+ModulemdModuleStreamV1 *
+modulemd_module_stream_v1_parse_yaml (yaml_parser_t *parser, GError **error);
```

Comment:
Could you make this accept a `SubdocumentInfo` like `Translation` and `Defaults` take?
e.g. https://github.com/fedora-modularity/libmodulemd/blob/2.0-dev/modulemd/v2/modulemd-translation.c#L457

Reply:
A patch for this is in https://github.com/sgallagher/libmodulemd/pull/1